frontend/javascripts/oxalis/model/helpers/nml_helpers.js

```diff
@@ -167,12 +179,39 @@ function serializeMetaInformation(
   ]);
 }
 
-function serializeBoundingBox(bb: ?BoundingBoxType, name: string): string {
-  const serverBoundingBox = convertFrontendBoundingBoxToServer(bb);
-  if (serverBoundingBox != null) {
-    const { topLeft, width, height, depth } = serverBoundingBox;
-    const [topLeftX, topLeftY, topLeftZ] = topLeft;
-    return serializeTag(name, { topLeftX, topLeftY, topLeftZ, width, height, depth });
+function serializeTaskBoundingBox(boundingBox: ?BoundingBoxType, tagName: string): string {
+  if (boundingBox) {
```

Comment:
```suggestion
  if (boundingBox != null) {
```

Reply:
Why do you think the `!= null` is needed? The expressions should be equal as the bounding box type cannot be evaluated to false if it exists., right? A bounding box is not a string or number that implicitly can be evaluated as false by JavaScript.

Reply:
Oh, I just noticed that in the old version the check was also made with `!= null`. Thus I'll also use the longer version of the if-statement :+1:

Reply:
You're right, feel free to ignore this suggestion :)